pkg/kudoctl/cmd/init.go

```diff
@@ -0,0 +1,155 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+
+	manInit "github.com/kudobuilder/kudo/pkg/kudoctl/cmd/init"
```

Comment:
why do we call this manInit? 🤔 what does it mean?

Reply:
go has issue with `init` as it is special :(... I have changed the name to `cmdInit`